wrappers/c/zxing.h

```diff
@@ -0,0 +1,143 @@
+#ifndef _ZXING_C_H
+#define _ZXING_C_H
+
+/*
+ * <stdbool.h>
+ *  bool
+ *
+ * <stdint.h>
+ *  uint8_t
+ */
+#include <stdbool.h>
+#include <stdint.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/*
+ * ZXing/ImageView.h
+ */
+
+typedef void * zxing_ImageView;
```

Comment:
using `void *` is a bad idea as it completely losses type safety on the C side. Please have a look at the following sites for inspiration on how this might be handled differently: https://github.com/enricgalceran/cpplib_wrapper and https://www.teddy.ch/c++_library_in_c/

Reply:
Right, using `void*` is one of the things I wanted to avoid, but I didn't know how... It didn't occur to me to use `typedef struct foo * foo;` at the time, nor to conditionally `typedef` to "the right thing". Thanks for the references.